cli/web_worker.rs

```diff
@@ -77,11 +77,18 @@ pub struct WebWorker {
   event_loop_idle: bool,
   terminate_rx: mpsc::Receiver<()>,
   handle: WebWorkerHandle,
+  pub has_deno_namespace: bool,
 }
 
 impl WebWorker {
-  pub fn new(name: String, startup_data: StartupData, state: State) -> Self {
+  pub fn new(
+    startup_data: StartupData,
+    state: State,
+    maybe_name: Option<String>,
+    has_deno_namespace: bool,
+  ) -> Self {
     let state_ = state.clone();
+    let name = maybe_name.unwrap_or_else(|| "".to_string());
```

Comment:
FIXME: in `util.log` there will be no "source" specified if `name` is empty string - probably need to have "public" name that is `Worker.name` and "internal" name for debugging purposes (as well as thread name).